models/stg_microsoft_ads__campaign_daily_report.sql

```diff
@@ -33,6 +33,7 @@ final as (
         bid_match_type,
         delivered_match_type,
         top_vs_other,
+        budget_association_status,
```

Comment:
@fivetran-sheringuyen what do you think about aggregating across `budget_association_status` (so like summing the below metrics, _not_ including `budget_association_status` as a new column, and grouping by everything else) 

basically i'm wondering, is it valuable to have the budget status? my first thought is not really, but perhaps people would want to have that information to filter out `ended` budget records or something. what do ya think?

Reply:
I was definitely internally debating this - I have no strong feelings either way. There are a couple additional fields that are not already included in the transforms model, e.g. top_vs_other, delivered_match_type, etc, which could have the same proactive arguments made for it (add it in and let the customer decide to filter it out) or conversely we could also approach with a more reactive approach where we have customers feedback. The reason I decided to not include it is I feel like I'd have to also add the other fields as well. But definitely open for discussion..I wonder if Clay could give us some additional insight here when we ask him to test out the branch for your other PR? (hard deletes and search surrogate keys)

Reply:
ah i meant should we even have it in the staging model 

you have a good point with regards to consistency with the other fields (ie top_vs_other) though. we include these fields in the staging layer and then aggregate past them in the transform, so it feels right to do the same for budget status (so this is resolved for me)

Reply:
Oh, I think it has to be in the staging model in order to pass the uniqueness tests though